### DISCUSSION TITLE
Is there a bug or was there a bug with Macrophotos?

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#69] [Ideas]
Bonjour cher Collègue et Ami @AlZorba 
J'ai vu que sur le vieux thacer, on a sur cette page beaucoup de photos macro : https://thacer.archaiodata.com/ceram.php?ID=8017&THAnull&INV=null
alors que sur le nouveau, on n'en a pas : http://localhost:5173/ceram?ID=8017&THAnull&INV=null

Je pense que c'est le nouveau qui a raison, mais ne suis pas tres sur de moi. Qu'en penses-tu?

(en passant, il ne faudrait pas envoyer INV=null ou THAnull, mais rien. Je pense que ça vient de la generation d'url. Je fait un ticket pour ça https://github.com/archaiodata/thacer/issues/70)

**Comment by AlZorba** [comment]
la aussi, remédié avec ID unique ver 3.0.0
